.env.in

```diff
@@ -1,21 +1,13 @@
-KURS_PM_DATA_API_URI=http://localhost:3001/api/kurs-pm-data
-KURS_PM_DATA_API_KEY=[generated in kurs-pm-data-admin-api key for admin page]
-KURS_INFO_API_URI=https://api-r.referens.sys.kth.se/api/kursinfo
-KURS_INFO_API_KEY=[generated in kursinfo-api in stage key for public pages, just for image displaying and selling text]
-REDIS_URI=[redis connection string, azure]
+KURS_PM_DATA_API_KEY=[Available in Azure KeyVault]
+KURS_INFO_API_KEY=[Available in Azure KeyVault]
+REDIS_URI=[Available in Azure KeyVault]
 SESSION_SECRET=[generate session secret]
-SESSION_KEY=kurs-pm-data-admin-web.pid
-APPLICATIONINSIGHTS_CONNECTION_STRING=[Azure, Application insights, connection string, can be found in Overview]
-OIDC_APPLICATION_ID=<FROM ADFS>
-OIDC_CLIENT_SECRET=<FROM ADFS>
+OIDC_APPLICATION_ID=[Available in Azure KeyVault]
+OIDC_CLIENT_SECRET=[Available in Azure KeyVault]
 OIDC_TOKEN_SECRET=<Random string>
-OIDC_CONFIGURATION_URL=<not needed if localhost>
-OIDC_CALLBACK_URL=<not needed if localhost>
-OIDC_CALLBACK_SILENT_URL=<not needed if localhost>
-OIDC_CALLBACK_LOGOUT_URL=<not needed if localhost>
 # Connection Properties of UG Rest Api to fetch names of course staff
 UG_REST_AUTH_API_TOKEN_URI=https://<LOGIN_HOST>/adfs/
```

Comment:
UG_REST_AUTH_API_TOKEN_URI and UG_REST_API_URI are available in ref.parameters.json.
It would be great if we could add them into serverSettings as devDefaults.

Reply:
serverSettings line 118 and 124.